docs/server/configuration.md

```diff
@@ -281,26 +309,6 @@ There are two sections defined under each service heading:
 
```

Comment:
Need some help on this section.

Reply:
bindOnIP and BindOnLocalHost are mutually exclusive.

localhost is only useful for scenarios where the services only need to be reachable from the same system, this is a convenience instead of just supplying `127.0.0.1` to BindOnIP.